Cargo.toml

```diff
@@ -30,7 +30,7 @@ pp-rs = { git = "https://github.com/Kangz/glslpp-rs.git", optional = true, rev =
 [features]
 default = []
 dot-out = []
-glsl-in = ["pp-rs"]
+glsl-in = ["pp-rs", "codespan-reporting"]
```

Comment:
For context, WGSL links to it because WebGPU needs to report good errors.
Contrary, Naga's GLSL frontent is likely to be used in glsl-to-wgsl or glsl-to-spirv converters. These should prioritize code size (for WASM), so for GLSL frontend rich error reporting should be optional.